lib/repo_tools.pm

```diff
@@ -69,6 +70,50 @@ sub smt_mirror_repo {
     save_screenshot;
 }
 
+sub rmt_wizard {
+
+    my $SCC_Password = get_var('SCC_PWD');
+    my $SCC_User     = get_var('SCC_USER');
+
+    # install RMT and mariadb
+    zypper_call 'in rmt-server';
+    zypper_call 'in mariadb';
+
+    # check mysql status and config mysql for RMT
+    systemctl 'start mysql.service';
+    systemctl 'status mysql.service';
+    my $cmd = 'mysql -u root -p <<EOFF 
+GRANT ALL PRIVILEGES ON \`rmt\`.* TO rmt@localhost IDENTIFIED BY \'rmt\'; 
+FLUSH PRIVILEGES; 
+EOFF';
+    type_string "$cmd\n";
+    assert_screen("rmt-sqladmin-password", 40);
+    send_key 'ret';
+
+    # Modify rmt config file
+    assert_script_run "sed -i '/^[][ ]*username:[][ ]*\$/d' /etc/rmt.conf";
+    assert_script_run "sed -i '/^[][ ]*password/d' /etc/rmt.conf";
+    assert_script_run "sed -i '/adapter/i\ \\ \\ password: rmt' /etc/rmt.conf";
+    assert_script_run "sed -i '/scc/a\ \\ \\ password: $SCC_Password' /etc/rmt.conf";
+    assert_script_run "sed -i '/scc/a\ \\ \\ username: $SCC_User' /etc/rmt.conf";
+
+    # Start rmt server
+    systemctl 'start rmt';
+    systemctl 'status rmt';
+}
+
+sub rmt_mirror_repo {
+
+    my $repo_list = get_var('RMT_REPO') || "sle-module-legacy/15/x86_64";
+    assert_script_run "rmt-cli sync", 1800;
```

Comment:
I suggest to use single ticks by default in all cases where you do not want to evaluate perl variables. Also in other lines